tensorflow/lite/micro/tools/make/targets/xtensa_makefile.inc

```diff
@@ -97,4 +97,10 @@ ifeq ($(OPTIMIZED_KERNEL_DIR), xtensa)
   $(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels/xtensa/lstm_eval.cc \
   $(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels/xtensa/lstm_eval_hifi.cc \
   $(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels/xtensa/unidirectional_sequence_lstm.cc
+
+  # override KERNEL_OPTIMIZATION_LEVEL to enable higher performance
+  # Xtensa intrinsics.
+$(KERNEL_OBJDIR)$(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels/xtensa/decompress.o: $(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels/xtensa/decompress.cc
```

Comment:
Curious if it's not sufficient to put this in `MICROLITE_CC_KERNEL_SRCS`? I understand that a dedicated rule was  required when the decompression routine was in `micro_context.cc`, but I'm not sure if it's needed now.

Reply:
Cadence requires us to use special compile options just for this one file.  Using -O3 on all kernels do NOT work.